packages/appium/lib/cli/plugin-command.js

```diff
@@ -17,6 +17,12 @@ export default class PluginCommand extends ExtensionCommand {
     this.knownExtensions = KNOWN_PLUGINS;
   }
 
+  /**
+   * Install a plugin
+   *
+   * @param {PluginInstallOpts} opts
+   * @return {Promise<ExtRecord<PluginType>>}
```

Comment:
For consistency, let's use `@returns` everywhere

```suggestion
   * @returns {Promise<ExtRecord<PluginType>>}
```